source/request-as-event-emitter.ts

```diff
@@ -56,20 +56,6 @@ export default (options: NormalizedOptions): RequestAsEventEmitter => {
 
 		const handleResponse = async (response: http.IncomingMessage): Promise<void> => {
 			try {
-				/* istanbul ignore next: fixes https://github.com/electron/electron/blob/cbb460d47628a7a146adf4419ed48550a98b2923/lib/browser/api/net.js#L59-L65 */
-				if (options.useElectronNet) {
-					response = new Proxy(response, {
-						get: (target, name) => {
-							if (name === 'trailers' || name === 'rawTrailers') {
-								return [];
-							}
-
-							const value = (target as any)[name];
-							return is.function_(value) ? value.bind(target) : value;
-						}
-					});
-				}
-
```

Comment:
Can you leave this one? We shouldn't remove any fixes as that would be a breaking change.